jasmine/spec/inverted-index-test.js

```diff
@@ -0,0 +1,71 @@
+const books = require('./books.json')
+
+
+describe('InvertedIndex Class', () => {
+  beforeEach(() => {
+    this.invertedIndex = new InvertedIndex();
+    this.index = this.invertedIndex.createIndex('books', books);
+  });
+
+  describe('Constructor', () => {
+    it('can create inverted index instance', () => {
+      expect(typeof this.invertedIndex).toEqual('object');
+      expect(this.invertedIndex instanceof InvertedIndex).toBe(true);
+    });
+
+    it('has an indexes object to hold all indexes', () => {
+      expect(typeof this.invertedIndex.indexes).toEqual('object');
+    });
+  });
+
+ describe('GetWords', () => {
+    it('should return an array of words', () => {
+      expect(InvertedIndex.getWords(books[0].title))
+        .toEqual(['alice', 'in', 'wonderland']);
+    });
+
+    it('filters out symbols', () => {
+      expect(InvertedIndex.getWords('alice # in* Won@derland'))
+        .toEqual(['alice', 'in', 'wonderland']);
+    });
+  });
+
+  describe('CreateIndex', () => {
+    it('creates an index', () => {
```

Comment:
Question: Is the index created from the title or the test of each document?

Also, I do not see you calling the `createIndex` method here where you're testing the result.